### PR TITLE
LPS-156343, LPS-156327

### PR DESCRIPTION
**Pull request by thaiscabrals**

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -891,6 +891,19 @@ definition {
...
+
+				Type(
+					locator1 = "ObjectAction#INPUT_EXPRESSION",
+					value1 = "${inputExpression}");
+			}
+			else if ("${inputCondition}" == "false") {
+				ObjectAdmin.disableCondition();
+			}
+		}
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:906]: I think I remember seeing this in another PR from you. If you need these changes for this test as well, you should close the original PR, commit on top of it, then resend the PR.

**Review by timpak — CHANGES_REQUESTED** [review]
Good job with the test! Please see my comment and resend.

Also, please squash all of these changes into one commit to keep the commit history clean. You shouldn't be making commits with spacing or SF errors and fixing them in separate commits.

**Comment by evanilson-santana** [comment]
New PR [here](https://github.com/timpak/liferay-portal/pull/309)

**Pull request closed without merging**